index.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="src/style.css" />
+    <!-- Google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <!-- Google fonts -->
+    <!-- Font awesome -->
+    <script
+      src="https://kit.fontawesome.com/79b3da0f3c.js"
+      crossorigin="anonymous"
+    ></script>
+    <!-- Font awesome -->
+    <script defer src="./src/navPopup.js"></script>
+    <script defer src="./src/script.js"></script>
+    <title>CodnigCon</title>
+  </head>
+  <body>
+    <header class="header-section">
+      <nav class="navbar nav">
+        <i class="menu-icon fa-solid fa-bars mobile-only"></i>
+        <section class="nav-popup mobile-only">
+          <ul class="nav-popup-list">
+            <a href="#" class="nav-popup-list-link">
+              <li class="nav-popup-list-item active-page">> Home</li>
+            </a>
+            <a href="./src/pages/about.html" class="nav-popup-list-link">
+              <li class="nav-popup-list-item">> About</li>
+            </a>
+            <a href="#" class="nav-popup-list-link">
+              <li class="nav-popup-list-item">> Program</li>
+            </a>
+            <a href="#" class="nav-popup-list-link">
+              <li class="nav-popup-list-item">> Join</li>
+            </a>
+            <a href="#" class="nav-popup-list-link">
+              <li class="nav-popup-list-item">> Sponsor</li>
+            </a>
+            <a href="#" class="nav-popup-list-link">
+              <li class="nav-popup-list-item">> News</li>
+            </a>
+          </ul>
```

Comment:
- I think your design lacks the first navbar that's on behance. Can we include it. With this, we would have followed the design to a reasonable level. 

Here is what I can see compared to Behance👇🏽 
| Your Navbar |
| :---:   | 
|<img width="1009" alt="Screenshot 2023-01-04 at 10 32 05" src="https://user-images.githubusercontent.com/99246359/210592083-193396d8-fc99-4db9-a805-e74218426ee0.png">  |

|Behance Navbar |
| :---:   | 
|<img width="1183" alt="Screenshot 2023-01-04 at 10 34 27" src="https://user-images.githubusercontent.com/99246359/210592135-7e0e322b-6f87-4987-895f-42b0cbc3eabd.png">
  |

- Can we fix this?? 
⚠️  Kindly note that this should be applied to the about page as well. 🌸